smt/surrogate_models/krg_based.py

```diff
@@ -26,8 +26,10 @@
 )
 from scipy.stats import multivariate_normal as m_norm
 from smt.sampling_methods import LHS
-from smt.applications.mixed_integer import GOWER
-
+try : 
+    from smt.applications.mixed_integer import GOWER
```

Comment:
I did not catch this before, but `surrogate_models` must not depend on `applications`! `applications` depends on `surrogate_models` hence `GOWER` must be defined here.